src/site/_includes/components/Author.js

```diff
@@ -43,10 +43,10 @@ module.exports = ({
     <div class="w-author">
       <img
```

Comment:
What do you think about also wrapping the author image in an anchor? I'd suggest adding tabindex=-1 to it so we don't create too many tab stops for folks navigating via keyboard or switch device.

Reply:
Should this be added to cards as well? The second author link may be hard to click, but I figured I'd ask.

Reply:
yep i think it should. I agree the 2nd author one will be a little hard to click, but they can click on the name as well